app/static/js/grid_scripts.js

```diff
@@ -0,0 +1,15 @@
+document.addEventListener(
+    'DOMContentLoaded',
+    function () {
+        let all_days = document.getElementsByClassName('day'), i;
```

Comment:
Remove `, i`. Also, is this just a duplication of what you have in `load_weeks`?

Reply:
I entered all of the initial settings to a 'DOMContentLoaded' with the relevant settings functions.
(Its been added as part of the solution we discussed about: "repetitions in code")